config/initializers/simple_form.rb

```diff
@@ -10,7 +10,7 @@
     b.use :input, class: 'field'
   end
 
-  config.wrappers :vertical_form, tag: 'div', class: 'mb2', error_class: 'has-error' do |b|
+  config.wrappers :vertical_form, tag: 'div', class: 'mb3', error_class: 'has-error' do |b|
```

Comment:
We keep going back and forth on this one, wonder if we need another wrapper - login form uses `.mb3` but IDV form uses `.mb2`.